Homework8/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Homework8.Task1;
+using Homework8.Task2;
+
+namespace Homework8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            MassiveElement.ShowMassiveElement();
+            var exceptionGenerator = new ExceptionGenerator();
+            try
```

Comment:
Not all task conditions are done

Reply:
What task and which conditions are not done?
"Create a class that will contain the **method/property** that throws the given exception." Do I understand correctly, that either method or property should be created?